packages/functional-tests/lib/fixtures/standard.ts

```diff
@@ -138,8 +151,50 @@ export const test = base.extend<TestOptions, WorkerOptions>({
       ],
     });
   },
+
+  emails: async (
+    { target, pages: { login }, emailOptions },
+    use
+  ): Promise<void> => {
+    const emails = await Promise.all(
+      emailOptions.map(async (emailOptions) =>
+        login.createEmail(emailOptions.prefix)
```

Comment:
I'm not 100% sure about this&hellip; but I wonder if _this_ is where we can default the `prefix` to an empty string to make it a tiny bit DRYer.

```suggestion
        login.createEmail(emailOptions.prefix ?? '')
```

Now if the `emailOptions.prefix` is nullish it will just default to an empty string, as long as we make `prefix` optional above in the type definition.

Reply:
incorporated. Thanks!